test/torchaudio_unittest/common_utils/case_utils.py

```diff
@@ -96,8 +95,8 @@ def skipIfNoModule(module, display_name=None):
     return unittest.skipIf(not is_module_available(module), f'"{display_name}" is not available')
 
 
-skipIfNoSoxBackend = unittest.skipIf(
-    'sox' not in torchaudio.list_audio_backends(), 'Sox backend not available')
 skipIfNoCuda = unittest.skipIf(not torch.cuda.is_available(), reason='CUDA not available')
-skipIfNoSox = unittest.skipIf(not is_sox_available(), reason='Sox not available')
+skipIfNoSox = unittest.skipIf(
+    not is_sox_available() or
+    'sox' not in torchaudio.list_audio_backends(), reason='Sox not available')
```

Comment:
There is no need to update the logic. `'sox'` has been removed from the list of backends so `'sox' not in torchaudio.list_audio_backends()` is always `False`.